### ISSUE TITLE
Active UI does not keep focus when it is used outside the specified area

### ISSUE THREAD

**Issue by IRooc** [Wassimulator/CactusViewer#24]
When you are interacting with a UI element that allows sliding (GIF scrubber and Edit Hue, Saturation etc... when while dragging you move outside the area it looses focus and stops

Example:
![CactusViewer-UI-bug](https://github.com/Wassimulator/CactusViewer/assets/5981147/0264ce0f-a734-4e93-9a0b-c5aa89ee9a57)

This makes it harder to use those sliders, it would be nice if the active UI element keeps focus and you can drag outside the 'slider' region.

I build it on windows with commit acfcc00931750ae8f3ba2257e37478c9386c6ca2

Thanks for this it is a really nice and easy tool.

**Comment by Wassimulator** [comment]
you are absolutely right and this is something that a proper UI should support, I'll keep it in mind for the next update
